src/compressed_tensors/quantization/lifecycle/initialize.py

```diff
@@ -174,7 +174,10 @@ def _initialize_scale_zero_point(
         device = get_execution_device(module)
 
     # infer expected scale/zero point shape
-    expected_shape = 1  # per tensor
+    if quantization_args.strategy == QuantizationStrategy.TOKEN:
+        expected_shape = (1, 1)
+    else:
+        expected_shape = 1
```

Comment:
Where are you seeing failures?
The only relevant case is dynamic per token, which shouldn't be initializing anything so I dont think this change is required.